content/en/docs/tutorial/microservices-istio/_index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Learn Microservices using Kubernetes and Istio 
```

Comment:
Actually, I think this would be a good name, even if we move it to Examples.

Reply:
Fixed